java/testdata/java.yaml

```diff
@@ -3,3 +3,7 @@ commandTests:
   - name: java
     command: ["/usr/lib/jvm/java-8-openjdk-amd64/jre/bin/java", "-version"]
     expectedError: ['openjdk version "1\.8.*"']
+fileExistenceTests:
+  - name: certs
+    path: "/etc/ssl/certs/java/cacerts"
+    shouldExist: true
```

Comment:
Is it possible to have an app test this?  I'm not 100% clear on what this is needed for, but if it's just outbound SSL maybe hitting something that never goes down like `https://www.google.com`? :)

Reply:
Yep, added one that does exactly that.